### PR TITLE
fix: Issue #2138 - Angular build error

### PR DESCRIPTION
**Pull request by naticaceres**
## Description:
When Building an Angular app v12 and above that consumes okta-signin-widget and imports its css, the build fails with the following error:
`Error: undefined:5:49: missing '}'`
That error points to the compiled css that belongs to okta-signin-widget. Upon inspection, it corresponds to an uncompiled sass rule: `@use`.

When building the okta-signin-widget project, the compiled css shows the same error.

From a closer inspection at how scss files are compiled, and the scss rules used, the following problems surface:
  - the `@use` rules are making reference to files that no longer exist in the specified location, these references need to be fixed.
  - grunt-css is using a deprecated sass compiler: node-sass

This Pull request fixes the mentioned problems:
  - the `@use` rules routes are fixed to point to the intended file in its current location
  - dart sass was installed, grunt-scss bumped to its next minor version, and dart sass is configured in gruntfile.js according to grunt-scss official guidelines, using fibers package to speed up the compilation process

## PR Checklist

- [x] Have you verified the basic functionality for this change?

- [ ] Added unit tests? _- not applicable - this is a configuration and package update PR_

- [ ] Added e2e tests _- not applicable - this is a configuration and package update PR_

- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)? - _don't have access to see these_

### Screenshot/Video:

https://user-images.githubusercontent.com/10617214/129822348-a226e4e2-127f-4c7d-a8dc-196f467d9208.mp4

/--EDIT--/
in the video you can see me copy pasting the content from https://global.oktacdn.com/okta-signin-widget/5.9.2/css/okta-sign-in.min.css into http://csslint.net/ to validate css errors, and finding a bunch of them.
Then you can see me copy the compiled css from my fork into csslint.net, then getting 0 errors.

### Issue:

https://github.com/okta/okta-signin-widget/issues/2138

https://github.com/okta/okta-signin-widget/issues/2131

### CODE REVIEW

File: assets/sass/modules/_consent.scss [hunk 1]
```diff
@@ -85,7 +85,7 @@ $consent-light-text-color: #6e6e78; // Gray 600 in Odyssey
       width: 140px;
     }
 
-
+
```
Comment (denysoblohin-okta) [inline assets/sass/modules/_consent.scss:88]: Please revert adding whitespace

**Review by denysoblohin-okta — APPROVED** [review]
LGTM. Please fix whitespace to pass lint

File: assets/sass/modules/_consent.scss (hunk 1 above)
Comment (naticaceres) [reply assets/sass/modules/_consent.scss]: done

**Comment by naticaceres** [comment]
This PR fixes these two issues:

#2131  and  #2138

**Comment by deekoder** [comment]
@naticaceres Thanks for this PR. We will take a look and get back to you soon.

**Review by aarongranick-okta — APPROVED** [review]
internal ref: OKTA-421537

**Comment by aarongranick-okta** [comment]
was merged: https://github.com/okta/okta-signin-widget/pull/2179

**Pull request closed without merging**